lib/cloudregister/registerutils.py

```diff
@@ -2613,8 +2635,6 @@ def __remove_credentials(smt_server_name):
 def __remove_repo_artifacts(repo_server_name):
     """Remove the artifacts related to repository handling for a registration
     """
-
-    __remove_credentials(repo_server_name)
```

Comment:
This still breaks the idea that we have one function that removes everything related to repositories, someone is going to have to work really hard to convince me that this is a good idea.